apps/vmq_server/priv/vmq_server.schema

```diff
@@ -1470,3 +1470,66 @@
                                                                                   {default, prefer_local},
                                                                                   {datatype, atom}
                                                                                  ]}.
+
+%% @doc plugins.<name>.plugin = <plugin> enables the plugin <plugin>
+%% and associates it with the name <name>. Plugins are started in the
+%% order defined by the names given, i.e., the plugin with <name>
+%% 'plugin1' is started before the plugin with <name> 'plugin2'.
+%%
+%%
+%% Plugin specific settings are set via the plugin itself, i.e., to
+%% set the 'file' setting for the myplugin plugin, add a line like:
+%%
+%%
+%% myplugin.file = /path/to/file
+{mapping, "plugins.$name.plugin", "vmq_server.user_plugins", [
+                                                              {datatype, atom},
+                                                              {commented, "myplugin"},
+                                                              {include_default, "mypluginname"}
+                                                             ]}.
+
+%% @doc plugins.<name>.path defines the location of the plugin
+%% associated with <name>. This is needed for plugins that are not
+%% shipped with VerneMQ.
+{mapping, "plugins.$name.path", "vmq_server.user_plugins", [
+                                                            {datatype, directory},
+                                                            {commented, "/path/to/myplugin"},
+                                                            {include_default, "mypluginname"}
+                                                          ]}.
+
+%% @doc File based authentication plugin. Enabled by default. Comment
+%% out to disable.  
+{mapping, "plugins.plugin1.plugin", "vmq_server.user_plugins", [
```

Comment:
I started out calling it `plugins.p1.name`, but I changed it to `plugin` because I found that easier to document. I think I'd prefer `name` over `app`. Also, I could change the config name of the plugin to something else, so it would read something like: `plugins.1acl.name = vmq_acl` ? Would that be nicer to read?